src/mmrelay/plugins/health_plugin.py

```
@@ -106,7 +112,11 @@ def generate_response(self) -> str:
  {snr_line}"""
 
     async def handle_meshtastic_message(
-        self, packet, formatted_message: str, longname: str, meshnet_name: str
+        self,
+        packet: dict[str, object],
```

Ruff (0.14.10)

Unused method argument: `packet`

(ARG002)